.vscode/settings.json

```diff
@@ -8,6 +8,7 @@
     "cmake.configureSettings": {
         "BUILD_EXPERIMENTAL_FEATURES":true,
         "BUILD_STATIC_BINARIES":false,
+        "BUILD_OpenMP":true,
     },
     //"cmake.configureEnvironment": {
     //    "WX_CONFIG":"/Users/rohoua/work/software/wx/bin/wx-config",
```

Comment:
This seems specific to Alexis' configuration?  Perhaps it should be set to a generic path?

Reply:
Good point. It's commented out so has no effect, but might as well remove identifying information.